pkg/adapter/apiserver/config.go

```diff
@@ -68,4 +68,12 @@ type Config struct {
 	//
 	// +optional
 	Filters []eventingv1.SubscriptionsAPIFilter `json:"filters,omitempty"`
+
+	// SkippedPermissions is an experimental field that communicates that the
+	// ApiServerSource did not check if permissions were OK to setup
+	// watches on the required resources and namespaces. This changes the behaviour
+	// of the pod, so it does not keep trying for failed watches.
+	//
+	// +optional
+	SkippedPermissions bool `json:"skippedPermissions,omitempty"`
```

Comment:
I might be wrong but I think that this isn't filled when instantiating the Config struct

Reply:
When instantiating the `Config` construct this should be `false`, when creating the output for JSON it should be output as "false".